internal/conf/conftypes/conftypes.go

```diff
@@ -101,3 +102,33 @@ func (r *RawUnified) FromProto(in *proto.RawUnified) {
 func (r RawUnified) Equal(other RawUnified) bool {
 	return r.Site == other.Site && reflect.DeepEqual(r.ServiceConnections, other.ServiceConnections)
 }
+
+// Bedrock ModelIDs can either be just a <model_id> string as specified in https://docs.aws.amazon.com/bedrock/latest/userguide/model-ids.html. Or when using provisioned capacity a string like `<model_id>/<provisioned_capacity>` where the <provisioned_capacity> refers to the arn in https://docs.aws.amazon.com/bedrock/latest/APIReference/API_CreateProvisionedModelThroughput.html#API_CreateProvisionedModelThroughput_RequestSyntax
+type BedrockModelIdParts struct {
+	Model               string
+	ProvisionedCapacity string
+}
+
+func ParseBedrockModelId(modelId string) BedrockModelIdParts {
+	parts := strings.SplitN(modelId, "/", 2)
```

Comment:
As it turns out, `strings.SplitN` will return a slice with zero elements if given an empty string. So in some degenerate case this function will panic on line 116.
```suggestion
    if modelId == "" {
        return BedrockModelIdParts{}
    }
	parts := strings.SplitN(modelId, "/", 2)
```

https://go.dev/play/p/wlg3qDPUEmn

Also, re Golang naming conventions, it's more common to have the parameter name be `modelID` (capitalizing ID).